app/models/articles/feeds/variant_assembler.rb

```diff
@@ -0,0 +1,72 @@
+module Articles
+  module Feeds
+    # This module is responsible for assembling a named variant into the data structure used to
+    # build the feed query for a variant.
+    #
+    # @see .call
+    module VariantAssembler
+      # Assemble the named :variant based on the configuration of levers.
+      #
+      # @param variant [#to_sym,String,Symbol] the name of the variant we're assembling
+      # @param levers [Articles::Feeds::LeverCatalogBuilder] the available levers for assembly
+      # @param variants [Hash] the cache of previously assembled variants
+      #
+      # @return [Articles::Feeds::VariantQueryConfig]
+      def self.call(variant:, levers: Articles::Feeds.lever_catalog, variants: variants_cache)
+        variant = variant.to_sym
+        variants[variant] ||= begin
+          content = Rails.root.join("config/feed-variants/#{variant}.json").read
```

Comment:
[Suggestion] VERY minor, but it might be worth extracting this path into a constant for easier reference

Reply:
I've been working on the next pull request after this one, and it does include the directory as a constant.  (Which makes it easier to add a directory with broken configurations so we can test the unhappy paths)